pybossa/view/projects.py

```diff
@@ -271,13 +272,14 @@ def _description_from_long_description():
     project_repo.save(project)
 
     msg_1 = gettext('Project created!')
-    flash('<i class="icon-ok"></i> ' + msg_1, 'success')
-    flash('<i class="icon-bullhorn"></i> ' +
-          gettext('You can check the ') +
-          '<strong><a href="https://docs.pybossa.com">' +
-          gettext('Guide and Documentation') +
-          '</a></strong> ' +
-          gettext('for adding tasks, a thumbnail, using PYBOSSA.JS, etc.'),
+    flash(Markup('<i class="icon-ok"></i> {}').format(msg_1), 'success')
+    markup = Markup('<i class="icon-bullhorn"></i> {} ' +
+                    '<strong><a href="https://docs.pybossa.com"> {}' +
+                    '</a></strong> {}')
+    flash(markup.format(
+              gettext('You can check the '),
```

Comment:
This should be one message, otherwise it will broken in translations files and hard to understood by translators.